@i18n/ja/docs/concepts/tokens/decentralized-exchange/autobridging.md

```diff
@@ -15,9 +15,12 @@ XRP Ledgerの[分散型取引所](index.md)で、XRP以外の2種類の通貨を
 
 オートブリッジングは、あらゆる[OfferCreateトランザクション][]で自動的に行われます。[Paymentトランザクション](../../../references/protocol/transactions/types/payment.md)ではオートブリッジングはデフォルトでは _行われません_ が、path-findingにより同様の効果のある[パス](../fungible-tokens/paths.md)を検索できます。
 
+[![オートブリッジングによる直接注文と合成注文の組み合わせを示す図](/docs/img/autobridging.png)](/docs/img/autobridging.png)
+
+
 ## 関連項目
 
-- [Dev Blog: Introducing Autobridging](https://xrpl.org/blog/2014/introducing-offer-autobridging.html)
+- [開発者ブログ: オートブリッジング](https://xrpl.org/blog/2014/introducing-offer-autobridging.html)
```

Comment:
For cases like this, does it make sense to add a warning to the link that the article is not available in Japanese? like so:

```suggestion
- [開発者ブログ: オートブリッジング（英語）](https://xrpl.org/blog/2014/introducing-offer-autobridging.html)
```

Reply:
We do not indicate (English) even in other links to other untranslated content pages, so I think it is fine as it is.
Rather, at the top of pages that appear in English even though they are accessed via the /ja/ link, It would be good to have a notation such as “Not translated. Would you like to contribute?”

Reply:
I like the idea to add a notation on pages that appear in English when they are accessed via the /ja/ link. Created https://github.com/XRPLF/xrpl-dev-portal/issues/2828 to track this.